tensorflow/contrib/layers/ln_kernels/layer_norm_fused_grad_op.cc

```diff
@@ -0,0 +1,356 @@
+/* Copyright 2015 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Yes, I tried to put them in layers/kernels at first, but bazel would raise `crosses boundary of subpackages` error during build. After some googling, I found out that it's caused by the existing `BUILD` file in layers/kernels, but I couldn't figure out a way to write the build rules in this case, so I put the kernel codes in a separate folder as a temporary measure. It would be great if you could give me some guidance on how to write the build rules to resolve this.